src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java

```diff
@@ -39,4 +46,16 @@ public UploadResponse upload(List<MultipartFile> files) {
         }
     }
 
+    public byte[] download(String documentId) {
+        String serviceAuthorization = authTokenGenerator.generate();
+
+        try {
+            ResponseEntity<Resource> responseEntity =  documentDownloadClientApi.downloadBinary(DUMMY_OAUTH_2_TOKEN, serviceAuthorization, documentId);
```

Comment:
change the name of the token, please. Also it's not OAUTH, that's the service to service token :)

Reply:
s2s token we usually call it